### DISCUSSION TITLE
New Front-End Checklist website and rules are currently being build!

### DISCUSSION THREAD

**Discussion by thedaviddias** [thedaviddias/Front-End-Checklist#513] [General]
It has been few years since the first version of the Front-End Checklist! Web has changed in a lot of ways and the need for a better website and re-defining the rules has come!

For few weeks now, I have been working on improving the experience on a new version of the website to become easier to use but also a source of more content and knowledge.

But I need your help in suggesting ideas, improvements, features you would want to see being build. I'm trying to finalize a first version without a ton of features and iterate after this initial launch.

Feel free to suggest anything!

**Comment by deepeshbhatnagar** [comment]
Hi David,
I like the front-end checklist as being too compressive, looking forward to customizing messages using it on my local machine. Please suggest the way forward.

**Reply by thedaviddias** [reply]
Thanks @deepeshbhatnagar for your message. What exactly do you have in mind? What would be your need?

**Comment by cgwrench** [comment]
Hi David, thanks for a wonderful checklist - it's a great resource.

One item that jumps out to me that should be reviewed for the new checklist is:

> Embedded or inline CSS: Avoid at all cost embeding CSS in <style> tags or using inline CSS

Inlining critical CSS and deferring other styles can be super important for putting together a fast site.

Perhaps rather than forbidding inlined CSS, we could suggest that only critical CSS is inlined, and even then only a small amount. This might need a limit to be defined certain threshold (I'm not sure what an appropriate limit would be, I've seen less than 14 kb compressed for HTML + critical CSS used before).

**Comment by JorensM** [comment]
Hello, I have a suggestion in case this is still being worked on - offer links to reasoning/arguments/justifications behind list items where the reasoning might not be apparent.

**Reply by thedaviddias** [reply]
That's exactly one of the aspects I'm working one.

Each rule will have it's dedicated page with full explanations, reasoning etc. That should also serve as a way to learn more rather than just be "a rule".
